combat-trainer.lic

```diff
@@ -1158,6 +1205,94 @@ class SpellProcess
   end
 end
 
+class PetProcess
+  include DRC
+
+  def initialize(settings)
+    echo('New Pet Process') if $debug_mode_ct
+
+    @zombie = settings.zombie
+    echo("  @zombie: #{@zombie}") if $debug_mode_ct
+    
+    @is_present = false
+    @current_stance = nil
+    @current_behavior = nil
+  end
+
+  def execute(game_state)
+    check_zombie
+    if game_state.dismiss_pet?
+      game_state.next_clean_up_step
```

Comment:
Should this line and the next be done in the opposite order? I feel like we should dismiss the zombie before signaling to proceed with the next step, but I suspect this is functionally fine.